invenio_rdm_records/services/permissions.py

```diff
@@ -72,6 +73,10 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_read_files = [
         IfRestricted("files", then_=can_view, else_=can_all),
     ]
+    can_get_content_files = [
+        # preview is same as read_files
```

Comment:
This equivalences between preview/review and the new actions are taken from here <https://github.com/inveniosoftware/invenio-records-resources/commit/fd65e834e5426a789dd91e2544bdc52ae1ec1994#diff-13fd70056deb6848af5488fc2db9e690a05806d34b2b9333218e37bc60168d4b>. This means, we apply "same as before" but with the local files condition, resulting in:
```python
if local:
    same as before
else:
    only system process / superuser
```